openedx/features/edly/api/serializers.py

```diff
@@ -37,4 +53,22 @@ def get_site_data(self, obj):
             )
         )
         site_data['display_name'] = self.context['edly_sub_org_of_user'].lms_site.name
+        url = get_value_for_org(
+            self.context['edly_sub_org_of_user'].edx_organization.short_name,
+            'SITE_NAME',
+            default=''
+        )
+        protocol = 'https' if self.context['request'].is_secure() else 'http'
+        site_data['site_url'] = '{}://{}'.format(protocol, url) if url else ''
         return site_data
+
+    def get_mobile_enabled(self, obj):  # pylint: disable=unused-argument
+        """
+        Returns mobile_enabled flag
+        """
+        mobile_enabled = get_value_for_org(
+            self.context['edly_sub_org_of_user'].edx_organization.short_name,
+            'MOBILE_ENABLED',
+            default=False
```

Comment:
@zubair-arbi @iam-mhaseeb Should the mobile support be disabled by default?

Reply:
If there's no `MOBILE_ENABLED` flag in site_configuraton then it will be considered as disabled automatically.